CHANGELOG.md

```diff
@@ -12,6 +12,10 @@
 
 ### Bug Fixes
 
+- Fixed a bug where a wrong error message would be thrown on
+  naming variables starting with uppercase.
+  ([Arian Pourarian](https://github.com/Arian94))
```

Comment:
This isn't a bug, the previous error message was correct. We are adding additional detail to it here.

Reply:
Does it mean it's a feature? Or it is not worth mentioning it in the CHANGELOG file?

Reply:
It's a feature, adding new detail to an error message.